CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ HunterGate(URL "https://github.com/ruslo/hunter/archive/v0.19.89.tar.gz"
 # what? why would you get id of other useful build types? - Ellzey
 set(CMAKE_CONFIGURATION_TYPES "Debug;Release" CACHE STRING "Limited configurations" FORCE)
 
-project(libevent C)
+project(Libevent C)
```

Comment:
Actually it make sense to keep lowercase version and rename Hunter package. Windows and OSX is case insensitive for example.

Reply:
The problem is that none of their configs conform to the lower case name so this ends up breaking the hunter build. I could change the capitalization everywhere else, but that seems to contradict the intention of the author here.

Reply:
> that seems to contradict the intention of the author here

Okay, try to keep upstream API.